TAScheduler/UnitTests/tests.py

```diff
@@ -178,50 +189,827 @@ class TestCourseGetCrseInfo(TestCase):  # Randall
 
 
 class TestSectionGetID(TestCase):  # Joe
-    pass
+    section = None
+
+    def setUp(self):
+        tmpcourse = Course.objects.create(
+            course_id=100,
+            semester="fall 2023",
+            name="testCourse",
+            description="test",
+            num_of_sections=3,
+            modality="online",
+            credits=3
+        )
+        tmpcourse.save()
```

Comment:
I don't believe you need this command if you use the line Course.objects.create

Reply:
Ohhhhh, I was wondering why your code used those.